site/docs/connectors/integration/jdbc-integration-connector.md

```diff
@@ -0,0 +1,67 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
+
+--8<-- "snippets/content-status/tech-preview.md"
+
+# JDBC Integration Connector
+
+??? info "Connector details"
+    - Connector Category: [Integration Connector](/connectors/integration-connector)
+    - Hosting Service: [Database Integrator OMIS](/services/omis/database-integrator)
+    - Hosting Server: [Integration Daemon](/concepts/integration-daemon)
+    - Source Module: [jdbc-integration-connectors :material-github:](https://github.com/odpi/egeria-database-connectors/tree/main/jdbc-integration-connector){ target=gh }
+    - Jar File Name: `jdbc-integration-connectors.jar`
```

Comment:
The file I see, relative to root of the repo, is build/libs/jdbc-integration-connector-3.11-SNAPSHOT.jar

To follow the above pattern, I'd probably just say `jdbc-integration-connector-${version}.jar` but more detail on this should probably be in a README.md in the code repository - even if only saying what to run to build and where to find the file. IN fact adding at the top level makes sense.

Additionally
 - the current gradle build only publishes the postgres artifact, not jdbc - both as a build attachment, and as a maven artifact
  - the top level README needs to also refer to this connector
  
 But Both comments relate to the code repo -- and shouldn't stop us merging these doc updates, just a reminder of more work to do (will you update or create an issue @lcpopa ? )

Reply:
I do not think having the version in jar name applies here. This approach is in every other connector md file

Reply:
I suggest we move this question to a seperate issue. No need to stop the other useful updates due to this -- but it looks like we need more clarity about how someone uses a connector - ie where to get it from and what the file is called.

For example I know some people (such as myself - not production!) refer to the URL of a downloaded connector within our chart spec. That needs the exact filename - so knowing it contains version is critical. Even the gradle build themselves output  jars that contain the version.

I'll open an issue & close this discussion

Reply:
Thank you

Reply:
Opened #527 on the point about file versions
 * The name should still be singular (connector not connectors)
 * We should still add the basic info into the code repo readme - is that ok?

Reply:
Changed to 'connector'
There is some information in the repo [readme file](https://github.com/odpi/egeria-database-connectors/tree/main/jdbc-integration-connector), but it requires updating as a result of some ongoing changes. Any suggestion as to what should go in there, are much appreciated

Reply:
I was thinking:
 * postgres info needs to be pushed down from top level readme to the postgres directory (or we just use multiple top level sections - actually maybe that's easier)
 * The top level readme should ideally say what to run to build, and what is produced 
 
 I suggest we close this thread though so you can merge the docs in, and move over to the code repository

Reply:
Added issue on sorting readme -> https://github.com/odpi/egeria-database-connectors/issues/206